cmd/go-mutesting/main.go

```diff
@@ -434,6 +435,14 @@ func mutateExec(
 		debug(opts, "Execute built-in exec command for mutation")
 
 		diff, err := exec.Command("diff", "--label=Original", "--label=New", "-u", file, mutationFile).CombinedOutput()
+
+		changedLines := parser.ParseDiffOutput(string(diff))
```

Comment:
If you have empty changedLines, everything will crash.
And if there is something, 2 values ​​of some kind, then 0 will be overwritten.

Why return an array there? You can move this logic to the parser and decide there whether to return 0 or the real line number.